README.md

```diff
@@ -95,13 +95,10 @@ nix-env -iA nixos.alacritty
 zypper in alacritty
 ```
 
-### Pop!\_OS / Ubuntu
+### Pop!\_OS
 
-> If you're not running Pop!_OS, you'll have to add a third party repository first:
->
-> ```sh
-> add-apt-repository ppa:mmstick76/alacritty
-> ```
+> **Since this requires a Pop!\_OS specific PPA, it will not work on other Ubuntu
```

Comment:
Not sure that it's even worth mentioning Ubuntu here, since it's just for pop os already. I'd rather remove that part about Ubuntu, since there're many distros using `apt`.

Reply:
I've had it removed initially, but then added it because I thought that it might prevent some complaints.

But you're right that it should already be implicit by the removal of the `Ubuntu` from the header. If in the future people should frequently complain about that, we can still change it again to include this notice. Until then, better keep it short.